ldk/node/examples/selfTestLoop/package.json

```diff
@@ -29,6 +29,7 @@
   },
   "dependencies": {
     "@oliveai/ldk": "file:../..",
+    "moment": "^2.29.1",
```

Comment:
It's worth noting that momentjs is "deprecated" and in maintenance mode. They do urge people to adopt a different util for this FYI.

Reply:
I think I'm fine keeping it as moment for now, we can address it later